sherpa/plot/__init__.py

```diff
@@ -26,8 +26,30 @@
 and used, even when there is no available plot backend, it is just
 that no graphical display will be created.
 
-"""
+Which backend is used?
+----------------------
+
+When this module is first imported, Sherpa tries to import the
+backends installed with Sherpa in the order listed in
+`sherpa.plot.try_backends`. The first module that imports
+successfully is set as the active backend. The following command prints the
+name and the location on disk of that module::
+
+   >>> from sherpa import plot
+   >>> print(plot.backend)
+
```

Comment:
This now conflicts with the preceding paragraph (I tried to include it in this but as it is unchanged GH decided not to let me), which says

> Classes provide access to common plotting tasks, which is done by the plotting backend defined in the ``options.plot_pkg`` setting of the Sherpa configuration file.

So do no-longer allow the user to specify the backend?